app/(dashboard)/home/page.tsx

```diff
@@ -20,17 +22,35 @@ export default function Dashboard() {
 
   const BREAKPOINT = 1190;
 
+  const [showBanner, setShowBanner] = React.useState(false);
```

Comment:
I see you've already merged this so it's no biggie, but in the future we can probably abstract all of this into its own component to avoid code duplication and easily use the notification banner if we want to add it to other places

Reply:
I can open a new PR with this refactored